ci/docker_push.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ev
+want_push='false'
+for branch in "master" "1.0.0" "1.1.0"
+do
+   if [ "$TRAVIS_BRANCH" == "$branch" ]
+   then
+       want_push='true'
+   fi
+done
+# if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$want_push" == "true" ] && [ "$TEST_TYPE" == "normal" ]
+if [ 0 == 0 ]
+then
+    # docker login -e $DOCKER_EMAIL -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+    export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH; fi`
+    echo "TAG is $TAG"
```

Comment:
Any thoughts on tagging based on semantic versioning?
i.e., master branch can be "unstable", while a.b.c could be tagged with tags: envoy:a, envoy:a.b, envoy:a.b.c, and envoy:latest pointing to the a.b.c
However, if you are maintaining 1.0.0 and 1.1.0 branch for internal purposes, then please disregard the comment above.